can/io/trc.py

```diff
@@ -58,13 +55,16 @@ def __init__(
         """
         super().__init__(file, mode="r")
         self.file_version = TRCFileVersion.UNKNOWN
-        self.start_time: Optional[datetime] = None
+        self.start_time: float = 0
```

Comment:
This attribute is public, so changing the type could break user code. I'd suggest creating a private float attribute `_timestamp` and a property `timestamp`, that returns a datetime instance like before.

Otherwise this looks good to me.

Reply:
Good point! Changed it